ragna/_api/core.py

```diff
@@ -17,6 +20,13 @@ def app(config: Config) -> FastAPI:
     rag = Rag(config)
 
     app = FastAPI(title="ragna", version=ragna.__version__)
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=get_origins(config.ui.url),
```

Comment:
@pierrotsmnrd You added a patch that allowed connections from the API as well. Could you explain why? I've reverted here to UI url only and for me everything worked fine.

Reply:
I did it for the sake of indicating explicitly all allowed origins but yes, that's not necessary.